R/pkg/R/DataFrame.R

```diff
@@ -3715,3 +3715,25 @@ setMethod("rollup",
             sgd <- callJMethod(x@sdf, "rollup", jcol)
             groupedData(sgd)
           })
+
+#' alias
+#'
+#' @aliases alias,SparkDataFrame-method
+#' @rdname alias
+#' @name alias
+#' @examples
```

Comment:
I general it would nice to sweep all the files to make it more consistent. Capitalization, punctuation, examples. return and such.